package.json

```diff
@@ -14,13 +14,9 @@
     "clean:webui": "shx rm -rf assets/webui/",
     "build": "run-s build:*",
     "build:webui": "run-s build:webui:*",
-    "build:webui:download": "npx ipfs-or-gateway -c QmfQkD8pBSBCBxWEwFSu4XaDVSWK6bjnNuaWZjMyQbyDub -p assets/webui/",
+    "build:webui:download": "npx ipfs-or-gateway -c QmQz6PVJPqRZZtShVZbc8J3VdZE8pEd7p5RGujvGGUmSyx -p assets/webui/",
```

Comment:
This CID should always be the latest release of ipfs-webui unless it depends on a pre-release version.

Reply:
@olizilla it's from the linked PR (https://github.com/ipfs-shipyard/ipfs-webui/pull/1016) and needed, yes.